src/Psalm/Type/Atomic/CallableTrait.php

```diff
@@ -40,8 +40,8 @@ trait CallableTrait
      */
     public function __construct(
         $value = 'callable',
```

Comment:
```suggestion
        string $value = 'callable',
```
and corresponding `@param` can be dropped